### PR TITLE
Add acknowledgements for John and Tess

### PR DESCRIPTION
**Pull request by johannhof**
@johnwilander @hober let me know if anything is incorrect here :)

***
<a href="https://pr-preview.s3.amazonaws.com/johannhof/storage-access/pull/133.html" title="Last updated on Oct 25, 2022, 10:38 AM UTC (28704ef)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/privacycg/storage-access/133/c7abefa...johannhof:28704ef.html" title="Last updated on Oct 25, 2022, 10:38 AM UTC (28704ef)">Diff</a>

### CODE REVIEW

File: storage-access.bs [hunk 1]
```diff
@@ -370,6 +370,8 @@ The [=remote end steps=] are:
 
 <h2 id="acknowledgements" class="no-num">Acknowledgements</h2>
 
+This specification builds on the foundations created by former editors John Wilander, who invented the Storage Access API, and Theresa O’Connor, who wrote significant portions of the initial spec. We are grateful for their ideas and contributions.
```
Comment (annevk) [inline storage-access.bs:373]:
```suggestion
This specification builds on the foundations created by former editors John Wilander, who invented the Storage Access API, and Theresa O’Connor, who wrote significant portions of the initial text. We are grateful for their ideas and contributions.
```

**Review by annevk — APPROVED** [review]
I'll ping them to double check.

**Review by hober — APPROVED** [review]
Awwwwh, thanks!

**Review by johnwilander — APPROVED** [review]
Thanks!

**Pull request merged**